courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2565 @@
+{
```

Comment:
<p>Each Learning Objective should correspond to a TODO in the notebook below. These TODOs will be the Exercises that show up in the lab version of this solution.</p><p> </p><p>I would simplify the Learning Objectives to those relevant to this lab (i.e. feature engineering). So, I would have only</p><ol><li>Create an input pipeline using tf.data</li><li>Engineer features to create categorial and numeric features</li></ol><p> </p><p>Note Learning Objectives should be numbered. In the notebook below you can tie each TODO with the numbered Learning Objective. It's ok if one objective maps to multiple TODOs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>